src/rust/ide/view/graph-editor/src/component/node/output/port.rs

```diff
@@ -109,6 +113,42 @@ impl AllPortsShape {
     }
 }
 
+/// The length of the port shape's inner border (the part that touches the node)
```

Comment:
This is not a sentence, missing dot. The same applies to a lit of other lines in other docs. Please fix this in all new docs.